src/upnp.cpp

```diff
@@ -441,50 +436,6 @@ void upnp::on_reply(udp::endpoint const& from, span<char const> buffer)
 		return;
 	}
 
-	bool non_router = false;
-	if (m_ignore_non_routers)
-	{
-		auto const routes = enum_routes(m_io_service, ec);
```

Comment:
do you remember if `enum_routes` is still used in other parts of the code?

Reply:
yes, it's definitely used. In fact, I intend to make it mandatory for building libtorrent. See https://github.com/arvidn/libtorrent/pull/4249

Reply:
👍